test/functional/wallet_bumpfee.py

```diff
@@ -260,7 +260,9 @@ def test_unconfirmed_not_spendable(rbf_node, rbf_node_address):
 
 
 def test_bumpfee_metadata(rbf_node, dest_address):
-    rbfid = rbf_node.sendtoaddress(dest_address, Decimal("0.00100000"), "comment value", "to value")
+    assert(rbf_node.getbalance() < 49)
```

Comment:
I'd prefer to use the `assert_bla` helpers. Otherwise this is extremely difficult to debug in case of a failure.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15538#discussion_r262600574

> I'd prefer to use the assert_bla helpers. Otherwise this is extremely difficult to debug in case of a failure.

This is making an assertion about the test environment, and if it fails it probably means there is a bug in the test setup, not a bug in bitcoin. So I thought plain assert was reasonable here. Could go either way though. (For comparison, google gunit test framework has separate ASSERT_TRUE, ASSERT_EQ macros you're supposed to use for test assertions, and CHECK, CHECK_EQ macros you're supposed to use for normal code and test setup assertions).